cmd/command.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"os/exec"
+	"syscall"
```

Comment:
🤔 well that's a bit confusing. Especially since it's broken up into subpackages like `unix` and `windows` and I don't know how to conditionally use one. However, `Exec` doesn't even exist in `windows` and I'm pretty sure the current `syscall.Exec` doesn't work on Windows anyway...

So I guess I'll just switch to `sys/unix` for this PR. Good time to do it 👍

Reply:
Okay done 👍